lang/ja.yml

```
@@ -1,78 +1,78 @@
+DATABASE_LOCKED_1: "データベースがロックされています。最大15秒時間を頂く場合がございます。"
+DATABASE_LOCKED_2: "そのデータベースは既に使用されています。もう一度やり直してください。"
+DATABASE_LOCKED_3: "データベースロックを無効にするには、「database-lock: false」を設定してください。"
 DATABASE_LOCKED_4: "データベースロックを無効にすると、データが破損する可能性があります。"
 DATABASE_UNREACHABLE: "データベースにアクセスできません。データを破棄してシャットダウンします。"
 DEVELOPMENT_BRANCH: "開発ブランチが検出され、パッチスクリプトをスキップしました。"
-DIRT_BLOCK: "あなたの下にダートブロックを置きました。"
-DISABLE_SUCCESS: "成功！無効 {0}"
+DIRT_BLOCK: "あなたの下に土ブロックを置きました。"
+DISABLE_SUCCESS: "{0}の無効化に成功しました。"
 ENABLE_FAILED: "{0} を開始できませんでした。"
-ENABLE_SUCCESS: "これで {0} は有効になります！"
-ENJOY_COREPROTECT: "{0} を楽しんでいますか？私たちのDiscordに参加しましょう！"
-FINISHING_CONVERSION: "データ変換を終了します。お待ちください..."
-FINISHING_LOGGING: "データの記録が終了します。お待ちください..."
+ENABLE_SUCCESS: "{0} が有効化されました。"
+ENJOY_COREPROTECT: "{0} を楽しんでいますか？私たちのDiscordに参加しましょう!"
+FINISHING_CONVERSION: "データ変換を終了します。しばらくお待ちください..."
+FINISHING_LOGGING: "データのログへの記録を終了します。しばらくお待ちください..."
 FIRST_VERSION: "初期DB:{0}"
-GLOBAL_LOOKUP: "グローバルルックアップを実行するために半径を指定しないでください。"
-GLOBAL_ROLLBACK: "つかいます \"{0}「グローバルに {ロールバック|復元}"
-HELP_ACTION_1: "ルックアップをアクションに制限します。"
+GLOBAL_LOOKUP: "全体的な検索を実行するために半径を指定しないでください。"
+GLOBAL_ROLLBACK: "全体的な{ロールバック|復元}をするには、{0}を使用してください。"
+HELP_ACTION_1: "検索をアクションに制限します。"
 HELP_ACTION_2: "例:[a:block]、 [a:+block]、 [a:-block] [a:click]、 [a:container]、 [a:inventory]、 [a:item]、 [a:kill]、 [a:chat]、 [a:command]、 [a:sign]、 [a:session]、 [a:username]"
 HELP_COMMAND: "追加情報を表示します。"
 HELP_EXCLUDE_1: "ブロック/ユーザーを除外します。"
 HELP_EXCLUDE_2: "例:[e:stone]、 [e:Notch]、 [e:stone,Notch]"
 HELP_HEADER: "{0} ヘルプ"
 HELP_INCLUDE_1: "ブロック/エンティティを含めます。"
 HELP_INCLUDE_2: "例:[i:stone]、 [i:zombie]、 [i:stone,wood,bedrock]"
-HELP_INSPECT_1: "インスペクターを有効にすると、次のことができます。"
+HELP_INSPECT_1: "検査モードを有効にすると、次のことができます。"
 HELP_INSPECT_2: "左クリックして配置を確認します。"
 HELP_INSPECT_3: "右クリックして、隣接する削除を表示します。"
 HELP_INSPECT_4: "ブロックを配置して、削除を確認します。"
 HELP_INSPECT_5: "ブロックを液体に入れて配置を確認します。"
-HELP_INSPECT_6: "右クリックしてチェストトランザクションを表示します。"
+HELP_INSPECT_6: "右クリックしてチェスト入れ替えを表示します。"
 HELP_INSPECT_7: "ヒント:「/co i」を使用して犯人を追跡できます"
-HELP_INSPECT_COMMAND: "ブロックインスペクターを切り替えます。"
-HELP_LIST: "すべてのコマンドのリストを表示します。"
-HELP_LOOKUP_1: "コマンドショートカット。"
+HELP_INSPECT_COMMAND: "ブロック検査モードを切り替えます。"
+HELP_LIST: "全てのコマンドのリストを表示します。"
+HELP_LOOKUP_1: "コマンドのショートカットです。"
 HELP_LOOKUP_2: "追加のページを表示します。"
-HELP_LOOKUP_COMMAND: "ルックアップブロックデータ。"
-HELP_NO_INFO: "コマンドの情報 \"{0}\" 見つかりません。"
+HELP_LOOKUP_COMMAND: "ブロックデータを検索します。"
+HELP_NO_INFO: "コマンドの情報 「{0}」 が見つかりません。"
 HELP_PARAMETER: "詳細なパラメータ情報については、「{0}」を参照してください。"
-HELP_PARAMS_1: "を実行します {ルックアップ|ロールバック|復元}。"
-HELP_PARAMS_2: "ユーザーを指定して {ルックアップ|ロールバック|復元}。"
-HELP_PARAMS_3: "時間を指定します {ルックアップ|ロールバック|復元}。"
-HELP_PARAMS_4: "制限する {ルックアップ|ロールバック|復元} 半径に。"
-HELP_PARAMS_5: "制限する {ルックアップ|ロールバック|復元} アクションに。"
-HELP_PARAMS_6: "ブロック/エンティティをに含める {ルックアップ|ロールバック|復元}。"
-HELP_PARAMS_7: "ブロック/ユーザーをから除外する {ルックアップ|ロールバック|復元}。"
+HELP_PARAMS_1: "{検索|ロールバック|復元}を実行します。"
+HELP_PARAMS_2: "{検索|ロールバック|復元}をする際に特定ユーザーが起こした行動を巻き戻します。"
```

検索か復元の時でも、ユーザの行動をロールバックする。という意味になる。
It is misinterpreted to mean that the user's actions are rolled back, even when they are “retrieved” or “restored.”

```suggestion
HELP_PARAMS_2: "{検索|ロールバック|復元}するユーザを指定します。"
```